src/Core/Transform/GlobalRegistrationTwoImagesBSpline/Code.cxx

```diff
@@ -42,9 +41,9 @@ using PixelType = float;
 using ImageType = itk::Image<PixelType, ImageDimension>;
 
 static void
-CreateEllipseImage(ImageType::Pointer image);
+CreateEllipseImage(const ImageType::Pointer & image);
```

Comment:
We are modifying this image, so the `const` qualified is confusing.

Reply:
We are not modifying the pointer though.

Reply:
This was a clang-tidy warning.